markdown/core.py

```diff
@@ -54,17 +55,16 @@ def __init__(self, **kwargs):
         Creates a new Markdown instance.
 
         Keyword arguments:
-
-        * `extensions`: A list of extensions.
-            If an item is an instance of a subclass of `markdown.extension.Extension`, the  instance will be used
-            as-is. If an item is of type string, first an entry point will be loaded. If that fails, the string is
-            assumed to use Python dot notation (`path.to.module:ClassName`) to load a `markdown.Extension` subclass. If
-            no class is specified, then a `makeExtension` function is called within the specified module.
-        * `extension_configs`: Configuration settings for extensions.
-        * `output_format`: Format of output. Supported formats are:
-            * `xhtml`: Outputs XHTML style tags. Default.
-            * `html`: Outputs HTML style tags.
-        * `tab_length`: Length of tabs in the source. Default: 4
+            extensions (list[str | Extension]): A list of extensions.
+                If an item is an instance of a subclass of `markdown.extension.Extension`, the instance will be used
+                as-is. If an item is of type string, first an entry point will be loaded. If that fails, the string is
+                assumed to use Python dot notation (`path.to.module:ClassName`) to load a `markdown.Extension` subclass.
+                If no class is specified, then a `makeExtension` function is called within the specified module.
+            extension_configs (dict[str, dict[str, Any]]): Configuration settings for extensions.
+            output_format (str): Format of output. Supported formats are:
+                * `xhtml`: Outputs XHTML style tags. Default.
+                * `html`: Outputs HTML style tags.
```

Comment:
I think we'll need a blank line before the items for Markdown to parse it correctly.